src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -504,6 +510,116 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 		}
 	};
 
+	const getSortMapByUserType = useCallback((): MetadataSortMap[] => {
+		const isPublicCollection = !hasAccessToVisitorSpaceOfObject;
+
+		if (!isNil(user)) {
+			return ANONYMOUS_ACTION_SORT_MAP();
+		}
```

Comment:
shouldn't this be the inverse?
```suggestion
		if (isNil(user)) {
			return ANONYMOUS_ACTION_SORT_MAP();
		}
```

Reply:
Right 😅